scripts/earthengine/test_data/sample_fires_event_pipeline_config.py

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
We might want to document this (later PR).  I'm not sure if someone other than Ajai can setup event configs :)

Reply:
I'll add a separate config.py with details of each stage config.